docs/attribute_guidelines.md

```diff
@@ -1,25 +1,35 @@
-The following document aims to describe which parsable attributes of a parser class represent which semantic piece of a given news article.
-Consistency between publishers and parsers is a main goal, please report any cases you deem to be inconsistent with this document. 
+The following document aims to describe which parsable attributes of a parser class represent which semantic piece of a
+given news article.
+Consistency between publishers and parsers is a main goal, please report any cases you deem to be inconsistent with this
+document.
 If you want to contribute a parser to this library, please ensure that these attributes are named consistently.
 
+**_NOTE:_** There are certain utility functions to aid you with parsing which can be found
+under `fundus/parser/utility.py`.
+We *highly* recommend using them.
+You can find the corresponding utility function in the following table.
```

Comment:
This line could be improved upon: "The following table lists the attributes and includes the name of the corresponding utility function"

Reply:
Done